doc/release-notes/release-notes-1.0.15.md

```diff
@@ -0,0 +1,9 @@
+Notable changes
+===============
+
+UTXO memory accounting
```

Comment:
Put these in `doc/release-notes.md`, which gets inserted into the correct release notes file at release time. This also means you can just cherry-pick upstream commits directly, and their release notes will go into the correct place.

Reply:
Ok, done.